src/routes/results.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+class Results extends React.Component {
+  render() { //I'm not sure how to get the spacings right for this. Right now it looks kinda ugly.
+    return (
+      <View
+        style={styles.search}>
+        <Text style={styles.textWithMargin}>That is a tin can!</Text>
+        <Text style={styles.textWithMargin}>Recycle it</Text> 
```

Comment:
You can put the icons next to the text, I already have those in the repo. You'll probably want to group the icon and related text in a View together, with the heading text out on its own.